### ISSUE TITLE
Footer app icons URLs links should have role or label for screen readers

### ISSUE THREAD

**Issue by maxvisser** [teamforus/forus#1773]
![image](https://user-images.githubusercontent.com/10818702/93761670-7754a880-fc0e-11ea-9412-808bf5fcf6d8.png)
![image](https://user-images.githubusercontent.com/10818702/93761683-7c195c80-fc0e-11ea-8d54-a0eb28a52a39.png)
